workhorse.make

```diff
@@ -1077,6 +1077,9 @@ ifeq (,$(USE_SO_ATTRIBUTES))
 else
 	@$(ECHO) "Can't build product_files$(EXEEXT) with USE_SO_ATTRIBUTES."
 endif
+	@$(CP) --preserve md5sum$(EXEEXT) $(prefix)/third_party/bin
+	@[ -z "$(compiler_runtime_files)" ] \
```

Comment:
Please rebase this branch on the latest master, this shouldn't be necessary any longer since 19c09468c.